modules/vm/variables.tf

```diff
@@ -80,4 +85,9 @@ variable "node_count" {
   description = "Number of GraphDB nodes to deploy in ASG"
   type        = number
   default     = 3
+
+  validation {
+    condition     = var.node_count % 2 == 1 && var.node_count >= 3 && var.node_count <= 7
```

Comment:
We plan on adding support for single node deployment with this Terraform. Add a validation for an **odd** amount of nodes.

Reply:
Let's do odd numbers in general, but bigger than 1 for now. When we add support for a single node, we'll remove the "> 1" check. Running it 1 node now will just not work.

Reply:
I have changed it to be only odd numbers bigger than 1. Please keep in mind that now 9, 11 and etc. are available entries.